applications/nublado/values.yaml

```diff
@@ -348,6 +348,16 @@ hub:
   # controls how the database password is managed.
   internalDatabase: true
 
+  # -- Resource limits and requests for the Hub
+  # @default -- See `values.yaml`
```

Comment:
I don't understand what I did wrong here such that I'm getting the actual resources rather than "See `values.yaml`" in the `README.md`.